lago/utils.py

```diff
@@ -421,6 +421,10 @@ def deepcopy(original_obj):
     """
     if isinstance(original_obj, list):
         return list(deepcopy(item) for item in original_obj)
+    elif isinstance(original_obj, collections.OrderedDict):
+        return collections.OrderedDict(
+            (key, deepcopy(val)) for key, val in original_obj.items()
+        )
```

Comment:
Just replacing it with the stdlib copy.deepcopy does not seem to work, tests are failing for me after the change, will investigate:
```
not ok 22 basic.full_run: start and stop many vms one by one
# (from function `helpers.equals' in file tests/functional/helpers.bash, line 76,
#  from function `helpers.run_ok' in file tests/functional/helpers.bash, line 62,
#  in test file tests/functional/basic.bats, line 301)
#   `helpers.run_ok "$LAGOCLI" \' failed
# RUNNING:lago --prefix-name multi_vm init --template-repo-path /home/dcaro/Work/lago-project/lago/tests/functional/fixtures/basic/template_repo.json --template-repo-name local_tests_repo --template-store /home/dcaro/Work/lago-project/lago/tests/functional/fixtures/basic/repo_store --skip-bootstrap /home/dcaro/Work/lago-project/lago/tests/functional/fixtures/basic/suite_multi_vm.yaml
# --output--
# current session does not belong to lago group.
# @ Initialize and populate prefix: 
#   # Initialize prefix: 
#     * Create prefix dirs: 
#     * Create prefix dirs: Success (in 0:00:00)
#     * Generate prefix uuid: 
#     * Generate prefix uuid: Success (in 0:00:00)
#     * Create ssh keys: 
#     * Create ssh keys: Success (in 0:00:00)
#     * Tag prefix as initialized: 
#     * Tag prefix as initialized: Success (in 0:00:00)
#   # Initialize prefix: Success (in 0:00:00)
#   # Create disks for VM lago_functional_tests_vm02: 
#     * Create disk root: 
#       - Template local_tests_repo:lago_functional_tests:v1 not in cache, downloading
#     * Create disk root: Success (in 0:00:00)
#   # Create disks for VM lago_functional_tests_vm02: Success (in 0:00:00)
#   # Create disks for VM lago_functional_tests_vm01: 
#   # Create disks for VM lago_functional_tests_vm01: ERROR (in 0:00:00)
# @ Initialize and populate prefix: ERROR (in 0:00:00)
# Error occured, aborting
# Traceback (most recent call last):
#   File "/usr/lib/python2.7/site-packages/lago/cmd.py", line 810, in main
#     cli_plugins[args.verb].do_run(args)
#   File "/usr/lib/python2.7/site-packages/lago/plugins/cli.py", line 180, in do_run
#     self._do_run(**vars(args))
#   File "/usr/lib/python2.7/site-packages/lago/log_utils.py", line 615, in wrapper
#     return func(*args, **kwargs)
#   File "/usr/lib/python2.7/site-packages/lago/cmd.py", line 182, in do_init
#     do_bootstrap=not skip_bootstrap,
#   File "/usr/lib/python2.7/site-packages/lago/prefix.py", line 816, in virt_conf_from_stream
#     do_bootstrap=do_bootstrap,
#   File "/usr/lib/python2.7/site-packages/lago/prefix.py", line 930, in virt_conf
#     template_store=template_store,
#   File "/usr/lib/python2.7/site-packages/lago/prefix.py", line 836, in _prepare_domains_images
#     template_store=template_store,
#   File "/usr/lib/python2.7/site-packages/lago/prefix.py", line 858, in _prepare_domain_image
#     template_store=template_store,
#   File "/usr/lib/python2.7/site-packages/lago/prefix.py", line 886, in _create_disks
#     template_store=template_store,
#   File "/usr/lib/python2.7/site-packages/lago/prefix.py", line 463, in _create_disk
#     with LogTask("Create disk %s" % spec['name']):
# KeyError: 'name'
# ---
# "1" == "0"
```